docs/spacing.md

```diff
@@ -0,0 +1,108 @@
+# Spacing
+
+Careful spacing is key for the user to understand connections and hierarchies between elements, and for an interface to feel pleasant.
+
+## Scale
+
+Primer follows a base-8 scale, as it allows for significant flexibility when defining a variety of spacing values within an interface. 
+
+The possible values are: 0, 4, 8, 16, 24, 32, 40, 48
+
+For very small components where 8 pixel spacing is too large, you can use 4 pixels. 
+
+## Responsive breakpoints
+
+You can define different spacing values for different breakpoints, so the interface can better adapt to the available space.
+
+`[Example: component with larger margin around it in large screen, vs smaller margin in small screen]`
```

Comment:
Todo: Add example images

Reply:
Todo first: confirm this section is still correct/relevant.

Reply:
You could reference the general spacing of the entire page? Padding/margin around most pages reduces at smaller screen sizes. I can try to think of some other examples, too.

Reply:
thanks @colinkeany that's a good idea for now, do you know a page that would be better to use as an example? maybe something with not too much going on? 🤔

Reply:
The issues page might be a good option? https://github.com/issues